umh-core/pkg/service/redpanda/redpanda.go

```diff
@@ -775,6 +488,13 @@ func (s *RedpandaService) AddRedpandaToS6Manager(ctx context.Context, cfg *redpa
 	// Add the S6 FSM config to the list of S6 FSM configs
 	s.s6ServiceConfigs = append(s.s6ServiceConfigs, s6FSMConfig)
 
+	// Also add the redpanda monitor to the S6 manager
+	if err := s.metricsService.AddRedpandaMonitorToS6Manager(ctx); err != nil {
+		if !errors.Is(err, redpanda_monitor.ErrServiceAlreadyExists) {
```

Comment:
maybe add a comment that this action is supposed to be idempotent, so if the service already exists, it will continue